region/lowernorfair/west.json

```diff
@@ -965,7 +1013,18 @@
                     {"heatFrames": 3000}
                   ],
                   "note": "Expects two IBJs; one to break a block, then another one to get back up.",
-                  "devNote": "Needs another strat for a bomb-above IBJ. But the heat frames need to be checked by someone who can execute it."
+                  "devNote": "Needs strats for variations of IBJs, mainly for heatframe count." 
+                },
+                {
+                  "name": "Instant Morph Bomb Placement.",
+                  "notable": false,
+                  "requires": [
+                    "h_canNavigateHeatRooms",
+                    "h_canUseMorphBombs",
+                    "can2HighWallMidAirMorph",
+                    {"heatFrames": 1100}
+                  ],
+                  "note": "Requires one instant morph to place the bomb in position, and a second to land in the hole created by the explosion."
```

Comment:
This one isn't making sense to me. Is it used to bomb the bottom bomb block then fall and get back up into that spot? I'm assuming this is getting up with canPreciseWalljump, which is now included in the can2HighWallMidAirMorph

Reply:
I can clarify it, but yeah.  Jump up, place bomb, fall.  Jump up again.
And the walljump is included in `can2HighWallMidAirMorph`

For instant morphs you are aiming for the lower portion of a wall tile for more time to do the trick.  Thats why that change was made.  

Probably should have `canConsecutiveWalljump` too